src/storybook/changelog.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta } from '@storybook/blocks'
 
-<Meta title="Typography" />
+<Meta title="ChangeLog" />
```

Comment:
Can you add a line to the 4.1.0 release notes to say you upgraded prettier too please